cocotb/handle.py

```diff
@@ -36,7 +36,7 @@
 import os
 
 if "COCOTB_SIM" in os.environ:
-    import simulator
+    from cocotb import simulator
 else:
     simulator = None
```

Comment:
My hunch is that now that `simulator` is in the right place, this can just be
```
from cocotb import simulator
```

Reply:
Yes. Some places need a few lines more. a sperate PR is easier.